modules/shared-utils/src/cljdoc/util.clj

```diff
@@ -131,14 +131,20 @@
 ;;                                (-> cljdoc-edn :pom :version))))))
 
 
-(defn gh-owner [gh-url]
-  (second (re-find #"^https*://github.com/([^\/]+)/" gh-url)))
+(defn scm-owner [scm-url]
+  (get (re-find #"^https*://(github|gitlab).com/([^\/]+)/" scm-url) 2))
 
-(defn gh-repo [gh-url]
-  (second (re-find #"^https*://github.com/[^\/]+/([^/]+)" gh-url)))
+(defn scm-repo [scm-url]
+  (get (re-find #"^https*://(github|gitlab).com/[^\/]+/([^/]+)" scm-url) 2))
```

Comment:
Is there a benefit to using `get` over `second` here? I find the `second` a bit easier to parse visually since it doesn't have the additional argument.

Reply:
Oh nevermind, just realized that this is essentially `third` now. Ignore this please 👍